### DISCUSSION TITLE
MathJax/Markdown only transforms first paragraph

### DISCUSSION THREAD

**Discussion by steplee** [community/community#30565] [Other Feature Feedback, Questions, & Ideas] [answered]
On Linux with both chrome/firefox I am getting a bug where only the first set of contiguous lines has math expressions (surrounded by '$') transformed by mathjax.
MathJax renders all text in dollar signs until a double line-break, then no longer does.

Example:
https://gist.github.com/steplee/cd9df5db3427bf8ffe5af4083bdf7991

Note: ````math ...``` still works even at the end, so I think it has to do with parsing the dollar signs after any linebreaks.

**Comment by gititGoro** [comment]
I'm getting the same issue. It seemed to go wrong around August 23. I've also started a discussion with this demonstration gist

https://gist.github.com/gititGoro/b3ca31572976468991f46a71ccbba376

**Comment by nschloe** [comment, marked as answer]
This seems to have been fixed now.

**Reply by steplee** [reply]
Yes, it does.
